client/naturalcrit/homePage/homePage.less

```diff
@@ -127,6 +129,7 @@
 		//.animateAll(3s);
 	}
 }
+/* prettier-ignore-start */
```

Comment:
Not sure why we reference prettier here. We don't have prettier in this project that I am aware of.